src/features/buy-now/BuyNowModal.tsx

```diff
@@ -6,10 +6,12 @@ import { Modal } from '@zero-tech/zui/components';
 
 import { BuyNow } from './BuyNow';
 
+import styles from './BuyNow.module.scss';
+
 export interface BuyNowModalProps extends BasicModalProps {}
 
 export const BuyNowModal: FC<BuyNowModalProps> = ({ ...modalProps }) => (
-	<Modal {...modalProps}>
+	<Modal className={styles.Modal} {...modalProps}>
```

Comment:
Should be able to handle this one in `ConnectWallet` - will add overhead having to add `display: flex` to every Modal.

Reply:
Ah yes good shout. Updated!